src/usr/local/www/system_advanced_network.php

```diff
@@ -203,6 +210,16 @@
 ))->setHelp('By default, if IPv6 is configured and a hostname resolves IPv6 and IPv4 addresses, '. 
 	'IPv6 will be used. If this option is selected, IPv4 will be preferred over IPv6.');
 
+$section->addInput(new Form_Checkbox(
+	'ipv6dontcreatelocaldns',
+	'IPv6 DNS entry',
+	'Do not generate local IPv6 DNS entries for LAN interfaces',
+	$pconfig['ipv6dontcreatelocaldns']
+))->setHelp('If a LAN interface\'s IPv6 configuration is set to Track, and the tracked interface loses connectivity, '.
+    'it can cause connections to this firewall that were established via IPv6 hostname to fail. This can happen '.
+	'unintentionally when accessing the system by hostname, since by default both IPv4 and IPv6 entries are added '.
+	'to /etc/hosts. Enabling this option prevents those IPv6 records from being created.');
```

Comment:
I would not mention `/etc/hosts` here -- That isn't necessarily where they're going or where they're pulled from (unbound doesn't use it). Say "entries are added for resolution by DNS" or some variation of that.